Bottom: ccca746d86ec160d6578a1ed6a1a4b61632132c0
Top:    6a7bff00f7c34e8ea682b98b92f9da6effbe9e08
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 19:35:18 +0000

Refresh of allow-running-of-action-script

---

diff --git a/src/main.c b/src/main.c
index 162a0e0..e3c164f 100644
--- a/src/main.c
+++ b/src/main.c
@@ -996,12 +996,6 @@ main (int argc, char *argv[])
       LayerStringToLayerStack (Settings.InitialLayerStack);
     }
 
-  if (gui->printer || gui->exporter)
-    {
-      gui->do_export (0);
-      exit (0);
-    }
-
   /*    FIX_ME
      LoadBackgroundImage (Settings.BackgroundImage); */
 
@@ -1034,6 +1028,12 @@ main (int argc, char *argv[])
       hid_parse_actions (Settings.ActionString, 0);
     }
 
+  if (gui->printer || gui->exporter)
+    {
+      gui->do_export (0);
+      exit (0);
+    }
+
 #if HAVE_DBUS
   pcb_dbus_setup();
 #endif
